services/aur/aur.service.js

```diff
@@ -118,6 +119,36 @@ class AurVotes extends BaseAurService {
   }
 }
 
+class AurPopularity extends BaseAurService {
+  static category = 'rating'
+
+  static route = { base: 'aur/popularity', pattern: ':packageName' }
+
+  static openApi = {
+    '/aur/popularity/{packageName}': {
+      get: {
+        summary: 'AUR Popularity',
+        description: 'Arch linux User Repository Popularity',
+        parameters: pathParams({ name: 'packageName', example: 'dropbox' }),
+      },
+    },
+  }
+
+  static defaultBadgeData = { label: 'popularity' }
+
+  static render({ popularity }) {
+    return {
+      message: metric(popularity),
+      color: floorCountColor(popularity, 0.5, 2.5, 5),
```

Comment:
Do these threshold numbers come from anywhere in particular, or are they just based on vibes?

Reply:
They are based on my own vibes.

Reply:
Having done a bit of reading on how this is calculated/used on the AUR wiki, I don't think I have a better suggestion for a criteria